tests/FSharpVSPowerTools.Core.Tests/GoToDefinitionTests.LoadDirective.fs

```diff
@@ -0,0 +1,80 @@
+﻿module GoToDefinitionTests
+
+open System.IO
+open Microsoft.FSharp.Compiler.Range
+open Microsoft.FSharp.Compiler.SourceCodeServices
+open NUnit.Framework
+open FSharpVSPowerTools.Pervasive
+open FSharpVSPowerTools.UntypedAstUtils.HashDirectiveInfo
+
+[<Literal>]
+let dataFolderName = __SOURCE_DIRECTORY__ + "/../data/"
+type dataFolder = FSharp.Management.FileSystem<dataFolderName>
+
+let checker = FSharpChecker.Create()
+
+let canonicalizeFilename filename = Path.GetFullPathSafe filename //(new FileInfo(filename)).FullName
+
+let getAst filename = 
+    let contents = File.ReadAllText(filename)
+
+      // Get compiler options for the 'project' implied by a single script file
+    let projOptions = 
+        checker.GetProjectOptionsFromScript(filename, contents)
```

Comment:
I think we should use `LanguageService` instead of using `FSharpChecker` directly. Most of out core tests rely on `LanguageService`.

In `LanguageService.GetScriptCheckerOptions`, you could use `FSharpCompilerVersion.FSharp_3_1` to ensure compatibility with F# 3.1.